public/components/datasources/components/manage/accelerations/acceleration_table.tsx

```diff
@@ -95,13 +134,13 @@ export const AccelerationTable = ({
     }
   }, [accelerationsLoadStatus]);
 
-  const handleRefresh = () => {
+  const handleRefresh = useCallback(() => {
     console.log('Initiating refresh...');
     if (!isCatalogCacheFetching(accelerationsLoadStatus)) {
       setIsRefreshing(true);
       startLoadingAccelerations(dataSourceName);
     }
-  };
+  }, [startLoadingAccelerations, isCatalogCacheFetching, setIsRefreshing]);
```

Comment:
Do we need all of these dependencies here? Would `accelerationsLoadStatus` be the only dependency, since it's the only thing that will change?